app/views/exports/scrna_core_cell_extraction_tagging_csv_for_custom_pooling.csv.erb

```diff
@@ -0,0 +1,19 @@
+<%
+  csv_array = [['Source Well', 'Volume to add to pool', 'Dest. well', 'Number of samples', 'Tag index', 'Tag 2 index']]
+  @plate.wells_in_columns.each do |well|
+    # We want to skip wells that are empty or have multiple aliquots with different tag indices
```

Comment:
I assume you've copied this check from the Cardinal version, but I don't understand how that situation (multiple aliquots with different tag indices) would ever arise...